internal/services/logic/logic_app_workflow_resource.go

```diff
@@ -79,6 +253,16 @@ func resourceLogicAppWorkflow() *pluginsdk.Resource {
 				},
 			},
 
+			"state": {
```

Comment:
does this make sense to change to 
```suggestion
			"enabled": {
```
as a boolean to match other resources?

Reply:
Updated